draft-ietf-ppm-dap-taskprov.md

```diff
@@ -611,7 +617,7 @@ Finally, the Helper completes the request as usual, deriving the VDAF
 verification key for the task as described in {{vdaf-verify-key}}. During an
 aggregation job, for any report share that does not include the `taskprov`
 extension with an empty payload, the Helper MUST mark the report as invalid
-with error "invalid_message" and reject it.
+with error "invalidMessage" and reject it.
```

Comment:
This is actually correct as is. Here we mean that the `ReportError` should be set to the `invalid_message` variant. `invalidMessage` refers to an error variant that would be stuffed in a JSON response when aborting a DAP request.